README.md

```diff
@@ -0,0 +1,258 @@
+    
+    from aodndata.myfacility.handlers import MyFacilityHandler
+    
+    TEST_ROOT = os.path.join(os.path.dirname(__file__))
+    GOOD_NC = os.path.join(TEST_ROOT, 'myfacility_good.nc')
+    NOT_A_NETCDF = os.path.join(TEST_ROOT, 'not_a_netcdf_file.nc')
+    
+    
+    class TestMyFacilityHandler(HandlerTestCase):
+        """It is recommended to inherit from the HandlerTestCase class (which is itself a subclass of the standard
+           unittest.TestCase class). This provides some useful methods and properties to shortcut some common test
+           scenarios.
+        """
+        #This is a "boilerplate" method that must appear in each test case in order to correctly inherit from the HandlerTestCase class
+        def setUp(self):
+            # set the handler_class attribute to your handler (as imported above)
+            self.handler_class = MyFacilityHandler
+            super(TestMyFacilityHandler, self).setUp()
+    
+        def test_good_file(self):
+            # we expect this to succeed, so if the handler experiences an error, it is considered a
+            # "failed test"
+            handler = self.run_handler(GOOD_NC)
+            pass
+    
+        def test_good_file_with_compliance_check(self):
+            # we also expect this to succeed, since the test file is known be CF compliant
+            handler = self.run_handler(GOOD_NC, check_params={'checks': ['cf']})
+            pass
+    
+        def test_bad_file(self):
+            # we expect this to fail with a 'ComplianceCheckFailedError' exception, since it's not actually
+            # a NetCDF file since we expect this to be a failure, we use run_handler_with_exception to
+            # invert the expected result, so that it treats a success as an undesired outcome, and therefore
+            # a "failed test"
+            handler = self.run_handler_with_exception(ComplianceCheckFailedError, NOT_A_NETCDF)
+            pass
+
+    ```
+
+1. You can now test your handler by simply running the unit tests. There are several ways to run them in the IDE, but you can get fine-grained control over which tests are run by opening the test module, and right-clicking on the test class and clicking _'Run unittests for Unittests for test_handlers.TestMyFacilityHandler'_, or even the individual test methods. This enables the handler class itself to be largely configured
+and tested before leaving the IDE, and proceeding on to integration testing, to run the handler in a "deployed" context.
+
+1. For further documentation relating to available handler parameters, and how a handler class works, refer to the upstream [aodncore documentation](https://github.com/aodn/python-aodncore). 
+Handler parameters consist of a single positional parameter, which is always 'input_file', a 'config' object (set automatically in both unittests and when deployed), an optional Celery task parameter, set when run under a Celery task, and a series of keyword arguments to control
```

Comment:
... (unfinished sentence). "... a series of keyword arguments to control the behaviour of individual handling steps"?